activerecord/lib/active_record/store.rb

```diff
@@ -123,7 +120,21 @@ def store_accessor(store_attribute, *keys, prefix: nil)
 
       def _store_accessors_module # :nodoc:
         @_store_accessors_module ||= begin
-          mod = Module.new
+          mod = Module.new do
+            def self.stored_accessor_key(store_attribute, key, options)
```

Comment:
Why do we define the helper methods in each `_store_accessors_module`s?

Reply:
@kamipo, thanks for the review! The definition is there so that we can call the chunk of code in _store_accessors_module context, but it's true that each model that calls store/store_accessor defines the helper method inside the _store_accessors_module. Maybe I should just move that part of code directly into inside the module_eval block, unless you have any better suggestion. If you do, let me know :)